.ci_scripts/update_docs

```diff
@@ -31,11 +31,11 @@ if [ -n "$GH_TOKEN" ]; then
     cd ..
     mv docs-README docs/README
 
-    git diff
+    # git diff
 
     # Only commit the docs for commits on master.
     # GITHUB_HEAD_REF is empty on forks
-    if [[ "$GITHUB_REF" == "ref/heads/master" ]] && [[ "$GITHUB_HEAD_REF" == "" ]]; then
+    if [[ "$GH_REF" == "ref/heads/master" ]] && [[ "$GH_HEAD_REF" == "" ]]; then
```

Comment:
I think these need to be set in the workflow YAML using the `${{ .... }}` syntax?

Reply:
Ah I'm wrong, `GITHUB_REF` should be set: https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

Reply:
Maybe. I am having trouble with it so I put them in the yaml.